src/SourceBuild/content/eng/pipelines/ci.yml

```diff
@@ -77,6 +77,59 @@ extends:
       baseline:
         baselineFile: $(Build.SourcesDirectory)\.config\guardian\.gdnbaselines
 
+    containers:
+      almaLinuxContainer:
+        image: mcr.microsoft.com/dotnet-buildtools/prereqs:almalinux-8-source-build

```

Comment:
These image names are being duplicated with the variables defined in variables/vmr-build.yml. We should have a single list of image names defined so that we don't have to maintain both of these files.

Reply:
@ellahathaway, have you tried importing https://github.com/dotnet/sdk/blob/main/eng/pipelines/templates/variables/vmr-build.yml in this template and referencing the variables?  I remember you mentioning having troubles utilizing variables here but don't remember what you tried.

Reply:
I did try that. From what I remember, importing the variables did not work in this instance. I will try again for sanity sake, but I think the followup here is to add a comment saying that these values need to be kept up-to-date with the values in the variables template.

Reply:
Well... it worked? Not sure what I did the first time, but it's possible that I used macro syntax instead of template expression syntax. Either way, I'll open a PR to update the variables.